lang/defs.bzl

```diff
@@ -0,0 +1,24 @@
+def _generate_tables_impl(ctx):
+    args = ctx.actions.args()
+    args.add("--input", ctx.file.src)
```

Comment:
opt: can use single '-' in go flags '-input'

Reply:
Done.